.github/workflows/styler-actions.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  issue_comment:
+    types: [created]
+name: Commands
+jobs:
+  style:
+    if: startsWith(github.event.comment.body, '/style')
+    name: style
+    runs-on: macOS-latest
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+      - name: testing
+        run: echo '${{ steps.file_changes.outputs.files_modified}}'   
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: r-lib/actions/setup-r@master
+      - name: Install dependencies
+        run: |
+          Rscript -e 'install.packages(c("styler", "devtools"), repos = "cloud.r-project.org")'
+          Rscript -e 'devtools::install_version("roxygen2", version = "7.0.2", repos = "http://cran.us.r-project.org")'
+      - name: string operations
+        run: |
+          echo '${{ steps.file_changes.outputs.files_modified}}' > names.txt
+          cat names.txt | tr -d '[]' > changed_files.txt
+          text=$(cat changed_files.txt)
+          IFS=',' read -ra ids <<< "$text"
+          for i in "${ids[@]}"; do if [[ "$i" == *.R\" || "$i" == *.Rmd\" ]]; then echo "$i" >> files_to_style.txt; fi; done
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: artifacts
+          path: files_to_style.txt
+      - name: Style
+        run: for i in $(cat files_to_style.txt); do Rscript -e "styler::style_file("$i")"; done
+      - name: commit
+        run: |
+          git add \*.R
+          git add \*.Rmd
+          if [[ $(git diff --name-only --cached) != "" ]]; then  git commit -m 'automated syle update' ; fi
+      - uses: r-lib/actions/pr-push@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+
+  check:
+    needs: [style]
+    runs-on: ubuntu-latest
+    container: pecan/depends:develop
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: r-lib/actions/setup-r@master
+      - name : download artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: artifacts
+      - name : make
+        shell: bash
+        run: |
+          cut -d / -f 1-2 artifacts/files_to_style.txt | tr -d '"' > changed_dirs.txt
+          cat changed_dirs.txt
+          sort changed_dirs.txt | uniq > needs_documenting.txt
+          cat needs_documenting.txt      
+          for i in $(cat needs_documenting.txt); do make .doc/${i}; done
+      - name: commit
+        run: |
+          git config --global user.email "pecan_bot@example.com"
+          git config --global user.name "PEcAn stylebot"
+          git add \*.Rd
```

Comment:
Roxygen runs also have the potential to change the NAMESPACE and (hopefully rarely; only when the Roxygen version changes) DESCRIPTION files. I propose we handle those in a (near)future PR rather than this one, but the incentive is that should make it possible (if not the most convenient) to entirely skip Roxygen on your own machine.